inscriptions/ISic001670.xml

```diff
@@ -0,0 +1,225 @@
+                        <acquisition>Palermo, Museo Archeologico Regionale Antonino Salinas</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <note type="location">On display</note>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+             <p>Encoded following EpiDoc guidelines 9.2</p>
+             <xi:include href="../alists/ISicily-taxonomies.xml">
+                 <xi:fallback>
+                     <p>Taxonomies for ISicily controlled values</p>
+                 </xi:fallback>
+             </xi:include>
+         </encodingDesc>
+        <profileDesc>
+            <calendarDesc>
+                <calendar xml:id="julian">
+                    <p>Julian Calendar</p>
+                </calendar>
+            </calendarDesc>
+            <langUsage>
+                <language ident="en">English</language> 
+                <language ident="it">Italian</language> 
+                <language ident="grc">Ancient Greek</language> 
+                <language ident="la">Latin</language> 
+                <language ident="he">Hebrew</language> 
+                <language ident="phn">Phoenician</language>
+                <language ident="xpu">Punic</language>
+                <language ident="osc">Oscan</language> 
+                <language ident="xly">Elymian</language> 
+                <language ident="scx">Sikel</language>  
+                <language ident="sxc">Sikan</language>  
+            </langUsage>
+            <textClass>
+                <keywords scheme="http://www.eagle-network.eu/voc/typeins.html">
+                    <term ana="#function.abecedarium" ref="http://www.eagle-network.eu/voc/typeins/lod/112.html">abecedarium</term>
+                </keywords>
+            </textClass>
+        </profileDesc>
+        <revisionDesc status="draft">
+            <listChange>
+                    <change when="2021-02-11" who="#JP">Jonathan Prag created file from template, photograph and De Simone 2013</change>
+	</listChange>
+        </revisionDesc>
+    </teiHeader>
+    <facsimile>
+          <surface type="front">
+                    <graphic n="screen" url="ISic001670_tiled.tif" height="4912px" width="7360px">
+                        <desc>Photo J. Prag, courtesy Museo Archeologico Regionale Antonino Salinas</desc>
+                    </graphic>
+              <graphic n="print" url="ISic001670.jpg" height="4912px" width="7360px">
+                  <desc>Photo J. Prag, courtesy Museo Archeologico Regionale Antonino Salinas</desc>
+              </graphic>
+           </surface>
+    </facsimile>
+    <text>
+        <body>
+            <div type="edition" subtype="transliteration" xml:space="preserve" xml:lang="xpu-Latn" resp="#JP">
```

Comment:
I guess if the convention is to report the text in transliteration even among specialists then it is better to proceed in this direction and find a solution for the issue with the underdots. On the other hand, if there are Unicode characters for the letters of the alphabet, it would be nice to reproduce them as they are, as we do for Greek... But this needs to be asked to an expert so that the decision is not too far out of their editing habits.

Reply:
I basically agree with Valentina. However, since the aim of the digital edition would also be to overcome certain limits of the tradition of printed editions, my inclination would be to use unicode characters, thus avoiding conflicts with diacritics, but obviously we need to understand if this is also the inclination of the experts ...

Reply:
Thanks all. I've written to Maria Giulia Amadasi in the first instance (she is both lovely and the godmother of all Punic studies these days) and will then perhaps try out Jose Angel Zamora also.